fenix/app/src/main/java/org/mozilla/fenix/onboarding/JunoOnboardingFragment.kt

```diff
@@ -34,21 +38,31 @@ import org.mozilla.fenix.onboarding.view.telemetrySequenceId
 import org.mozilla.fenix.onboarding.view.toPageUiData
 import org.mozilla.fenix.settings.SupportUtils
 import org.mozilla.fenix.theme.FirefoxTheme
+import org.mozilla.gecko.search.SearchWidgetProvider
 
 /**
  * Fragment displaying the juno onboarding flow.
  */
 class JunoOnboardingFragment : Fragment() {
 
-    private val pagesToDisplay by lazy { pagesToDisplay(shouldShowNotificationPage(requireContext())) }
+    private val pagesToDisplay by lazy {
+        pagesToDisplay(
+            shouldShowNotificationPage(requireContext()),
```

Comment:
this is out of scope but would be nice is `shouldShowNotificationPage` was also `canShowNotificationPage`